src/FlaUI.Adapter.White/Extensions/AdapterExtensions.ClickableElement.cs

```diff
@@ -0,0 +1,111 @@
+﻿using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Exceptions;
+using FlaUI.Core.Logging;
+using FlaUI.Core.Tools;
+using System;
+using System.Drawing;
+using TestStack.White.InputDevices;
+using Wait = FlaUI.Core.Input.Wait;
+
+namespace TestStack.White.UIItems
+{
+    public static partial class AdapterExtensions
+    {
+        public static Mouse AttachedMouse = Mouse.Instance;
+        public static void ClickElement(this AutomationElement element)
+        {
+            if (element.FrameworkAutomationElement.IsKeyboardFocusable && !element.FrameworkAutomationElement.HasKeyboardFocus)
+            {
+                element.FocusElement();
+            }
+
+            var point = GetClickedPoint(element);
+            if (!point.IsEmpty)
+            {
+                AttachedMouse.Click(point);
+                element.WaitUntilInputIsProcessed();
+            }
+        }
+
+        public static void RightClickElement(this AutomationElement element)
+        {
+            var point = GetClickedPoint(element);
+            if (!point.IsEmpty)
+            {
+                AttachedMouse.RightClick(point);
```

Comment:
Any reason the `FocusElement` and `WaitUntilInputIsProcessed` is not added here as in `ClickElement`? It behaves differently and might cause confusion.